llm_on_ray/inference/inference_config.py

```diff
@@ -23,6 +23,8 @@ class ModelConfig(BaseModel):
     trust_remote_code: bool = False
     use_auth_token: Union[str, None] = None
     load_in_4bit: bool = False
+    torch_dtype: Union[str, None] = None
```

Comment:
IPEX has a `precision` config, vLLM has a `precision` config too. Here we are adding a new one. All should use this single config, right? @kira-lin @KepingYan @xwu99

Reply:
need to update the model yaml files.

Reply:
this torch_dtype is used in AutoModelForCausalLM.from_pretrained. It's different from IPEX one. torch_dtype can be left empty to use default type.

This ModelConfig is just kwargs of from_pretrained function. Ideally, users should be able to specify any key supported by from_pretrained, but they can also run it without specifying anything.